src/main/scala/coupledL2/L2Param.scala

```diff
@@ -132,7 +132,8 @@ case class L2Param(
   hasSramCtl: Boolean = false,
 
   // Enable new clint
-  EnablePrivateClint: Boolean = false
+  EnablePrivateClint: Boolean = false,
+  PrivateClintRange: AddressSet = AddressSet(0x38000000L, 0xFFFF)
```

Comment:
Do `PrivateClintRange` here and `TIMERRange` in the YML file need to be manually kept the same? Is there a better way to ensure that the two remain consistent?

If it cannot be ensured automatically, please add comments in both places to let other developers know that they must keep these two ranges consistent.